### ISSUE TITLE
1Password: Python & Fish shell

### ISSUE THREAD

**Issue by samip5** [royalapplications/toolbox#39]
Does Royal TSX execute Fish env too or how does it decide Python to use?

**Comment by lemonmojo** [comment]
@samip5 You can configure Dynamic Folder execution options under "Preferences - Dynamic Folder". Currently, we only support the Zsh and Bash shells.

**Comment by samip5** [comment]
> @samip5 You can configure Dynamic Folder execution options under "Preferences - Dynamic Folder". Currently, we only support the Zsh and Bash shells.

It's set to system shell, which in my case is Fish but doesn't appear to honor it. It should have note somewhere that says it will not honor anything else than those two even if system shell is something other than Zsh or Bash.

**Comment by lemonmojo** [comment]
@samip5 I admit that it's not very prominently placed but if you hover your mouse over the "i" icon to the left of the Shell selection box, it says exactly that.

**Comment by samip5** [comment]
> @samip5 I admit that it's not very prominently placed but if you hover your mouse over the "i" icon to the left of the Shell selection box, it says exactly that.

Oh, totally missed that text. It probably should be also under the "Evaluate Shell Profile", as that says "~/.profie, etc" which to me sounds like Fish too. ;)

**Comment by lemonmojo** [comment]
@samip5 You're right that it's not very visible right now and there's definitely room for improvement but the two settings are independent from each other and the info text below the "Evaluate Shell Profile" is very specific to that setting (and already very long). I'll check if I can squeeze the "i"'s text below the selection box but I fear localization makes that a very tight fit...

**Comment by samip5** [comment]
Oh, didin't even though of localization issues. I got it working by remembering to place my Bash equilevent add path things to .profile converted from fish's syntax. :)

Problem solved for now at least as I was wondering why it was taking the wrong python all the time. ;)

**Comment by lemonmojo** [comment]
@samip5 Glad you solved the actual issue! I'll close this for now but will keep looking into making the description for the shell selection more visible. Thx for bringing this to our attention!

cheers,
Felix
